### PR TITLE
feat ✨: Add storage-hub zombie tests 

### PR DESCRIPTION
**Pull request by timbrinded**
This PR adds zombienet Native support back to storage hub.

Includes:
- new script `downloadPolkadot.ts` for getting polkadot binaries from GH
- new oneliner for running local network: `zombie:run:native`
- New CI flow which runs a zombienet with storage hub using native provider

> [!IMPORTANT]  
> The new polkadot download script only works with linux because AFAIK there are no published macOS builds for polkadot node

### CODE REVIEW

**Review by snowmead — APPROVED** [review]
LGTM 👍🏼

File: test/README.md [hunk 1]
```diff
@@ -65,24 +68,21 @@ bun test
...
 
 ### Spawning ZombieNet
 
-> [!WARNING]
-> Currently ZombieNet doesn't work with SH due to how we make our chainspecs. WIP
+> [!INFORMATION]
+> Currently SH on k8 is having issue due to how we are generating chain specs, hence why we use native provider
 
-To launch a non-ephemeral ZombieNetwork, use:
+To launch a non-ephemeral ZombieNetwork by executing the following in: `/test` directory:
```
[the remark this answers is not in the document]
Comment (ffarall) [reply test/README.md:79]: Also, before this, add instructions for downloading polkadot binearies and making them executable.

**Review by ffarall — APPROVED** [review]
Approved with some minor comments on documentation

File: test/README.md [hunk 2]
```diff
@@ -65,24 +68,21 @@ bun test
 ### Running ZombieNet Tests
 
 ```sh
-bun zombie:test:latest
+bun zombie:test:native
 ```
 
 ### Spawning ZombieNet
```
[the remark this answers is not in the document]
Comment (timbrinded) [reply test/README.md]: done

**Pull request merged**